engine/src/main/scala/cromwell/logging/TerminalLayout.scala

```diff
@@ -22,25 +24,23 @@ object TerminalLayout {
   implicit class ColorString(msg: String) {
     def colorizeUuids: String = {
       "UUID\\((.*?)\\)".r.findAllMatchIn(msg).foldLeft(msg) {
-        case (l, r) =>
-          val color = if (Option(System.getProperty("RAINBOW_UUID")).isDefined)
-            Math.abs(17 * r.group(1).substring(0,8).map(_.toInt).product) % 209 + 22
-          else 2
```

Comment:
It's an undocumented feature not used anywhere which relies on multiple magic numbers